EngineLayer/Mods/Mods.txt

```diff
@@ -47,6 +47,12 @@ DI   HCD:C7 H11 N1 O1
 CF   H2 C2 O1
 DR   Unimod; 1.
 //
+ID   Lactylation
+TG   K
+PP   Anywhere.
+MT   Less Common
+CF   H4 C3 O2
```

Comment:
Should the number of hydrogens be 3, not 4? It looks like one is lost from nitrogen from lysine (with formal charge zero) with N-lactylation
![image](https://user-images.githubusercontent.com/16342951/89480201-5d443100-d75a-11ea-8c7e-6a27c1e4e760.png)

e.g. methyl groups are -CH3, but we record CH2 in this file because one is lost.

Reply:
Good point, thank you.